### PR TITLE
http -> https (for mixed content issue)

### PR DESCRIPTION
**Pull request by kiyoto**
@edsiper, do you think you can check?

image

### CODE REVIEW

**Comment by edsiper** [comment]
Not sure if is an issue, basically when fluentd site is loaded over https the debugger is claiming that some linked/external content is being retrieved over plain HTTP (no TLS/SSL)

My opinion is that gathering external/static assets from trusted sites over https will only add some unnecessary latency to the viewer of Fluentd site

File: views/layout.erb [hunk 1]
```diff
@@ -62,10 +62,10 @@
         <ul class="loginbar pull-right">
           <li><a href="/faqs">FAQs</a></li>
           <li class="devider"></li>
-          <li class="socialbutton slack"><a href="http://slack.fluentd.org"><img src="/images/slack_badge.png" width="55px" /></a></li>
+          <li class="socialbutton slack"><a href="//slack.fluentd.org"><img src="/images/slack_badge.png" width="55px" /></a></li>
```
Comment (edsiper) [inline views/layout.erb]: missing protocol for slack URL

File: views/layout.erb [hunk 2]
```diff
@@ -62,10 +62,10 @@
...
           <li class="devider"></li>
           <li class="socialbutton github">
-            <iframe src="http://ghbtns.com/github-btn.html?user=fluent&amp;repo=fluentd&amp;type=watch&amp;count=true" allowtransparency="true" scrolling="0" frameborder="0" height="25px" width="95px">
+            <iframe src="//ghbtns.com/github-btn.html?user=fluent&amp;repo=fluentd&amp;type=watch&amp;count=true" allowtransparency="true" scrolling="0" frameborder="0" height="25px" width="95px">
```
Comment (edsiper) [inline views/layout.erb]: missing protocol

**Comment by edsiper** [comment]
@kiyoto just reviewed, there are a couple of links without protocol (http://something -> //something)

File: views/layout.erb (hunk 1 above)
Comment (kiyoto) [inline views/layout.erb]: This is not missing protocol but making the resource protocol-dependent. See https://stackoverflow.com/questions/550038/is-it-valid-to-replace-http-with-in-a-script-src-http

File: views/layout.erb (hunk 2 above)
Comment (kiyoto) [inline views/layout.erb]: See above

**Comment by kiyoto** [comment]
@edsiper Thanks. I believe my PR title was misleading. I want to make sure that when the user is using https, everything loads consistently from https. Without that, the user will see the "broken shield" icon at the top right, which hurts a sense of trust.

image

**Comment by okkez** [comment]
@kiyoto Could you resolve conflicts?

See also #68

**Comment by ashie** [comment]
Already done at #73

**Pull request closed without merging**